providers/dns/googlecloud/googlecloud.go

```diff
@@ -22,9 +24,15 @@ type DNSProvider struct {
 }
 
 // NewDNSProvider returns a DNSProvider instance configured for Google Cloud
-// DNS. Credentials must be passed in the environment variable: GCE_PROJECT.
+// DNS. Project name must be passed in the environment variable: GCE_PROJECT.
+// A Service Account file can be passed in the environment variable:
+// GCE_SERVICE_ACCOUNT_FILE
 func NewDNSProvider() (*DNSProvider, error) {
 	project := os.Getenv("GCE_PROJECT")
+	saFile := os.Getenv("GCE_SERVICE_ACCOUNT_FILE")
+	if len(saFile) > 0 {
```

Comment:
Thanks for the quick change! One more nitpick 😅 
`if saFile, ok := os.LookupEnv("GCE_SERVICE_ACCOUNT_FILE"); ok {
}`